apps/privacy-panel/js/ala/exceptions.js

```diff
@@ -123,7 +123,7 @@ function(panels, BlurSlider, SettingsListener, SettingsHelper) {
       link.appendChild(name);
 
       if (itemData.type) {
-        var type = document.createElement('small');
```

Comment:
Did you mean to remove this? Isn't ```type``` undefined now which is accessed on line 127?